pkg/operator/nodecadaemon.go

```diff
@@ -12,6 +12,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	appsv1informers "k8s.io/client-go/informers/apps/v1"
 	corev1informers "k8s.io/client-go/informers/core/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
```

Comment:
the preference here is to use the smaller group client types.  See kubernetes/types/apps/v1 below.

Reply:
removed with the change to pass in the eventRecorder